docs/en_US/Compression/Pruner.rst

```diff
@@ -70,26 +59,10 @@ User configuration for Level Pruner
 
 ..  autoclass:: nni.algorithms.compression.pytorch.pruning.LevelPruner
 
-Tensorflow
-""""""""""
-
-..  autoclass:: nni.algorithms.compression.tensorflow.pruning.LevelPruner
```

Comment:
@liuzhe-lz do we still support tensorflow, at least provided one pruner?

Reply:
The tensorflow example was removed here: https://github.com/microsoft/nni/pull/3242/files#diff-8555e1a0ab0c25960a752bdb8741ae4de1d9ab10634740970a657a9ebff38c42
You have reviewed it 😼

Reply:
@liuzhe-lz Sorry for deleting by mistake 🥺 
Upload the TensorFlow example `naive_prune_tf.py`.